app/forms/provider_interface/rejections_wizard.rb

```diff
@@ -14,6 +14,10 @@ def rejection_reasons
                 :single_attribute_names,
                 :reasons,
                 to: :rejection_reasons
+
+      def selectable_reasons
+        reasons.reject(&:deprecated?)
+      end
```

Comment:
what do you think of doing this as a sort of fake scope? eg
```
@rejection_reasons ||= RejectionReasons.from_config.selectable
```
and then the filtering of deprecated reasons takes place in the `RejectionReasons` class rather than here.

Reply:
Done.

Reply:
@carlosmartinez Actually I'm reverting this, it's going to be messy to do it that way because of the weird way this class is delegating to a method on the other class that returns an array (so it'd have to instead return something smarter).